create-kubernetes-secret.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [[ $(kubectl get secrets -n openfaas-fn | grep secret-api-test-key | wc -l) -eq 0 ]]
+then
+    kubectl create secret generic secret-api-test-key --from-file=secret-api-test-key=${HOME}/secrets/secret_api_test_key.txt --namespace openfaas-fn
```

Comment:
You can create with `--from-literal`, a file isn't needed. You can also create from environmental variables if you want to create the value earlier.